### DISCUSSION TITLE
Cannot load Softbody

### DISCUSSION THREAD

**Discussion by yswi** [bulletphysics/bullet3#3390] [General]
Hi, I am having trouble with p.loadSoftBody() when loading external .obj files downloaded elsewhere.
Either the 'Cannot load Softbody' message is displayed, or Pybullet freezes like it is stuck in an infinite loop. Those .obj file works perfectly with p.createVisualShape(p.GEOM_MESH,  fileName = filename), but causes problem when used with loadSoftBody().

In detail, if I run this code below,
```
physicsClient = p.connect(p.GUI)
p.resetSimulation(p.RESET_USE_DEFORMABLE_WORLD)
p.setGravity(0,0,0)
p.setTimeStep(1e-3)
p.setRealTimeSimulation(0)

print("cpt1")
spatulaId = p.loadSoftBody( filename, basePosition = [0,0,0], scale = 1)
print("cpt2")
p.changeVisualShape(spatulaId, -1, flags=p.VISUAL_SHAPE_DOUBLE_SIDED)
print("cpt3")
```

the message that I get is 
```
Version = 4.1 INTEL-16.1.7
Vendor = Intel Inc.
Renderer = Intel(R) Iris(TM) Plus Graphics 655
b3Printf: Selected demo: Physics Server
startThreads creating 1 threads.
starting thread 0
started thread 0 
MotionThreadFunc thread started
cpt1
```
and stays there forever.

Did I miss something in my code?
Thank you.

**Comment by raghavauppuluri13** [comment]
I have this same issue.

**Comment by programme184** [comment]
Hi, have you ever solve this problem?
